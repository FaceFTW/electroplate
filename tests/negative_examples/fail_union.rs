use electroplate::Electroplate;

#[derive(Electroplate)]
union DummyUnion {
    f1: u32,
    f2: u32,
}
