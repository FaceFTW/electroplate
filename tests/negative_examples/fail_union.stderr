error: Electroplate does not support unions
 --> tests/negative_examples/fail_union.rs:4:1
  |
4 | / union DummyUnion {
5 | |     f1: u32,
6 | |     f2: u32,
7 | | }
  | |_^
